src/doblib/action.py

```diff
@@ -55,23 +55,28 @@ def load_action_arguments(args, actions=None):
         "`values` can be defined as a constant value or as dictionary which allows "
         "dynamic values. Following is possible:\n"
         "\n"
-        "  `field`: .. A field name to copy the value from\n"
-        "  `lower`: .. The lower bounds for randomized values°\n"
-        "  `upper`: .. The upper bounds for randomized values°\n"
-        "  `prefix`: .. Prefix to add for the new value°°\n"
-        "  `suffix`: .. Suffix to add for the new value°°\n"
-        "  `length`: .. Generate a random alphanumeric value of this length°°\n"
-        "  `uuid`: .. Generate a new uuid. Supported values are 1 or 4°°\n"
-        "  `choices`: .. List of values to pick a random value°°°\n"
+        "  `field`: .. A field name to copy the value from°\n"
+        "  `lower`: .. The lower bounds for randomized values°°\n"
+        "  `upper`: .. The upper bounds for randomized values°°\n"
+        "  `prefix`: .. Prefix to add for the new value°°°\n"
+        "  `suffix`: .. Suffix to add for the new value°°°\n"
+        "  `length`: .. Generate a random alphanumeric value of this length°°°\n"
+        "  `uuid`: .. Generate a new uuid. Supported values are 1 or 4°°°\n"
+        "  `choices`: .. List of values to pick a random value*\n"
+        "  `domain`: .. Domain to pick a random record from**\n"
+        "  `length`: .. Number of records to pick***\n"
         "\n"
         "  Additionally for Date or Datetime the specific parts of the value can be\n"
         "  replaced with a constant integer or a dict with `lower` and `upper` value.\n"
         "  Following attributes can be used as keys:\n"
         "    `year`, `month`, `day`, `hour`, `minute`, `second`"
         "\n"
-        "°   Only available for Integer, Float, Date or Datetime\n"
-        "°°  Only available for Char, Html, or Text\n"
-        "°°° Only available for Char, Html, Text or Selection\n",
+        "°   Only available for Integer, Float, Date, Datetime, Char, Html, Text or Selection\n"
+        "°°  Only available for Integer, Float, Date or Datetime\n"
+        "°°° Only available for Char, Html, or Text\n"
+        "*   Only available for Char, Html, Text or Selection\n"
+        "**  Only available for Many2one, Many2many\n"
+        "*** Only available for Many2many\n",
```

Comment:
This gets hard to read because the symbols are close together.

Idea: We use something `[]` and letters to mark specific categories like n for numeric fields, t for textual fields, s for Selection, o for Many2one, m for Many2many and d for Date/Datetime and write those categories down here.

e.g.:
```
`field`: .. A field name to copy the value from[n,d,t,s]
...
`length` : .. Generate a random alphanumeric value of this length [t] or number of records to pick [m]
```

Below:
```
`n` .. Numeric fields like Integer, Float, Monetary
`t` .. Textual fields like Char, Html, Text
`s` .. Selection fields
`o` .. Many2one fields
`m` .. Many2many fields
`d` .. Date and Datetime fields
```